src/map-feature.js

```diff
@@ -285,26 +285,30 @@ export class MapFeature extends HTMLElement {
   // native cs: used by FeatureLayer._geometryToLayer(...),
   //            the fallback cs for map-geometry if its cs attribute is not specified
   _getFallbackCS() {
-    let csMeta;
+    let csMeta, cs;
     if (this._parentEl.nodeName === 'MAP-LINK') {
       // feature attaches to link's shadow
-      csMeta =
-        this._parentEl.shadowRoot.querySelector('map-meta[name=cs]') ||
-        this._parentEl.parentElement.getMeta('cs');
+      csMeta = this._parentEl.shadowRoot.querySelector(
+        'map-meta[name=cs][content]'
+      );
+      let linkCS = this._parentEl._templateVars.values.find(
```

Comment:
This will only obtain a reasonable units value when the query tref contains a bbox={xmin},{ymin},{xmax},{ymax} parameter.  In the case of a GetTile request, the location inputs will refer to tilematrix coordinates {x} and {y} (maybe) which can't be relied upon to be the same units as the returned feature.  In that case the \<map-meta name="cs"> is required data. cc @AliyanH

Reply:
deleted